src/h3/_cy/geo.pyx

```diff
@@ -54,35 +54,27 @@ cpdef (double, double) cell_to_latlng(H3int h) except *:
     return coord2deg(c)
 
 
-cdef h3lib.GeoLoop make_geoloop(geos, bool lnglat_order=False) except *:
```

Comment:
You're no longer allowing `lng,lat` input? Or you've moved this handling elsewhere?

Reply:
The input hasn't changed. It's still the same list of lat/lng pairs.